SPECS-EXTENDED/rear/rear.spec

```diff
@@ -65,43 +65,14 @@ Requires: bc
 ### Optional requirement
 #Requires: cfg2html
 
-%if %{?suse_version:1}0
-Requires: iproute2
-### recent SUSE versions have an extra nfs-client package
-### and switched to genisoimage/wodim
-%if 0%{?suse_version} >= 1020
-Requires: genisoimage
-%else
-Requires: mkisofs
-%endif
-###
-%endif
-
 %if %{?mandriva_version:1}0
-Requires: iproute2
-### Mandriva switched from 2008 away from mkisofs,
-### and as a specialty call the package cdrkit-genisoimage!
-%if 0%{?mandriva_version} >= 2008
-Requires: cdrkit-genisoimage
-%else
-Requires: mkisofs
-%endif
 #Requires: lsb
```

Comment:
For what it's worth, [newer versions of the spec from Fedora](https://src.fedoraproject.org/rpms/rear/blob/rawhide/f/rear.spec) don't require any sort of `lsb` package.